scripts/release.sh

```diff
@@ -0,0 +1,23 @@
+# This script cuts and uploads a release JAR to
+# our google cloud storage.
+#
+# https://storage.googleapis.com/sourcegraph-for-bitbucket-server/latest.jar
+#
+# The dependencies are:
+# 1) atlassian sdk (for building jar)
+# 2) gcloud cli setup with sourcegraph-dev project (for uploading jar)
+#
+# This script is to be run in the root folder.
+#
+# ./scripts/release.sh <OLD VERSION> <NEW VERSION>
+#
+# ex: ./scripts/release.sh 1.1.0 1.2.0
+
```

Comment:
I think this should have at the very least a `set -e` at the top so it doesn't continue on if anything here fails.